docs/modules/getting-started/pages/push.adoc

```diff
@@ -0,0 +1,112 @@
+= AeroGear Service Push SDK
+
+AeroGear for Android provides support for integrating Firebase Cloud Messaging (FCM) with the AeroGear Unified Push Server.
+
+Push messaging is a mechanism for sending data to mobile clients from a remote server. For mobile applications, this reduces battery usage, removes polling code and provides for a better user experience as data changes are communicated in real time.
+
+Messages that FCM receives from an application server are sent to the Android device over a connection managed by the operating system. These messages are then extracted and passed to an application using services definied provided by the AeroGear library.
+
+== Adding dependency
+
+Add the push dependency to your application module
+
+----
+implementation "org.aerogear:aerogear-push:[version]"
```

Comment:
I was following the guide. Seems to be not updated.

https://github.com/aerogear/aerogear-android-sdk/blob/master/docs/modules/getting-started/pages/getting-started.adoc#bill-of-materials

Do not get me wrong. I'm really happy to see BOM go away - it's just still there.

Reply:
@wtrocki I see. Most of the docs are outdated. My next step will be updating all docs on Android SDK, to reflect the new changes not documented. I was only waiting to get @finp PR (https://github.com/aerogear/aerogear-android-sdk/pull/204) merged

Reply:
I don't think we've released the SDK since the bom was taken out so those docs aren't worthless yet.

But yeah we should flag an issue to update the docs

Reply:
@secondsun I will review/update all Android SDK docs after got it merged